libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -2754,6 +2774,7 @@
 			baseConfigurationReference = CEBA94EF1C1B5D4C00C0B2E0 /* SalesforceSDKCore-Dynamic-iOS-Debug.xcconfig */;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
What's not safe for extensions?

Reply:
`UIApplication.shared.open(settingsUrl)` in ScreenLockUIView.  The button that navigates the user to settings.  I could remove the button if this is a big problem.

Reply:
This might be a problem for some of  our apps that use extensions, like SApp, for instance.